Requirements.txt

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
```

Comment:
Hi Zeynep, 

For the requirements.txt file, we only put int library names for examples: 
tqdm
cachetools
bs4
...